sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -38,7 +38,7 @@ class SnackBarAction extends Component {
   }
 }
 
-class SnackBar extends Component {
+class SnackBar extends StatefulComponent {
```

Comment:
Note that just owning an AnimatedPerformance doesn't mean you have to be rebuilt. It just means you don't get thrown away if your parent is rebuilt.